tools/pytorch2torchscript.py

```diff
@@ -139,6 +139,8 @@ def parse_args():
     parser.add_argument(
         '--verify', action='store_true', help='verify the TorchScript model')
     parser.add_argument('--output-file', type=str, default='tmp.pt')
+    parser.add_argument('--num-classes', type=int, default=150,
```

Comment:
may need to update docs as well:
https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/useful_tools.md#convert-to-torchscript-experimental

Reply:
Fixed, my apologies, I had been looking for the doc and somehow missed it.